downgrade

```diff
@@ -4,6 +4,31 @@
 export TEXTDOMAIN=downgrade
 export TEXTDOMAINDIR=/usr/share/locale
 
+usage() {
+  cat <<EOF
+Usage: downgrade [option...] <pkg> [pkg...] [-- pacman_option...]
+       downgrade <pkg> [pkg...] [option...] [-- pacman_option...]
```

Comment:
I don't think we need both lines. If you're trying to indicate that you can interleave options with packages, this doesn't even convey that anyway (it just shows "all packages, all options" or "all options, all packages"). I think we can just show the first line and let the user assume interleave-ability.

Reply:
Alright, fixed.